components/home/section2.tsx

```diff
@@ -7,26 +7,25 @@ import Underline from 'components/_common/underline'
 
 const CARDS = [
   {
-    img: '/images/coming-home.jpg',
+    img: '/bankless-website/images/coming-home.jpg',
     title: 'Departments',
     text: 'Departments are operation-critical units of BanklessDAO without which the DAO cannot function properly.',
     href: '/about-us/departments',
```

Comment:
Interestingly, the `href`'s in the `CARDS` list DON'T need the `/bankless-website` prefix, else they end up wrong `/bankless-website/bankless-website/about-us/departments`. :shrug: No clue why, but it's fixed now.